Vagrantfile

```diff
@@ -7,8 +9,17 @@ Vagrant.configure("2") do |c|
   c.vm.box_url = "https://atlas.hashicorp.com/ubuntu/boxes/trusty32/versions/14.04/providers/virtualbox.box"
   #c.vm.box_url = "https://atlas.hashicorp.com/chef/boxes/ubuntu-14.04-i386/versions/1.0.0/providers/virtualbox.box"
   c.vm.synced_folder ".", "/vagrant", disabled: true
+  c.vm.boot_timeout = 1800
+
   c.vm.provider :virtualbox do |p|
-    p.customize ["modifyvm", :id, "--memory", "128"]
+    p.customize ["modifyvm", :id, "--nictype1", "Am79C973"]
```

Comment:
Let's add a comment and reference @AndrewPashkin's comment regarding the nic type here:
http://help.appveyor.com/discussions/problems/1247-vagrant-not-working-inside-appveyor#comment_39277805